### ISSUE TITLE
Connectivity matrix based on rows (features)?

### ISSUE THREAD

**Issue by sdoerstling** [renozao/NMF#93]
Is there any way to extract the connectivity or consensus matrix for a multi-run nmf object that is based on the rows rather than the columns (samples)? I want to be able to extract the basis W matrix; will transposing the input matrix allow the clustering to be reversed and allow the H matrix to be the same as the untransposed W matrix?

Thank you!

**Comment by renozao** [comment]
Yes though you will have some variation in the result since the algorithm
will not start/stop at the same stage.
Beware that W usually has a large number of rows, which could make the
computation of the connectivity matrix in memory slow/impossible.
